docs/UserGuide.md

```diff
@@ -154,6 +158,73 @@ Example output:
 Balance: 3790.00
 ```
 
+### Viewing Watchlist: `watchlist`
+
+View your current watchlist with stocks that you are interested in
```

Comment:
maybe can put on what exchange the watchlist is displaying

Reply:
yes it should be included in the output already